src/emqx_slow_subs/emqx_message_latency_stats.erl

```diff
@@ -17,17 +17,18 @@
 -module(emqx_message_latency_stats).
 
 %% API
--export([ new/1, new/2, update/3
-        , check_expire/4, latency/1]).
+-export([ new/1, update/3, check_expire/4, latency/1]).
+
+-export([get_threshold/0, update_threshold/1]).
 
 -define(NOW, erlang:system_time(millisecond)).
 -define(MINIMUM_INSERT_INTERVAL, 1000).
--define(MINIMUM_THRESHOLD, 100).
+-define(MINIMUM_THRESHOLD, 500).
```

Comment:
Maybe two macros, one for minimum another for default.
 We can default to 500ms but should allow even lower config when requested by user